libs/shared/ui/src/lib/components/buttons/button-icon/button-icon.tsx

```diff
@@ -31,6 +31,9 @@ export interface ButtonIconProps {
   type?: 'button' | 'submit' | 'reset'
 }
 
+/**
+ * @deprecated This should be migrated to the <Button /> component
+ */
 export function ButtonIcon(props: ButtonIconProps) {
```

Comment:
Not sure if we want to maintain two buttons components, but I see on radix-theme that there are an `IconButton` as well https://www.radix-ui.com/themes/docs/components/icon-button

Reply:
Hum not sure also, radix doesn't have a button group component neither.